frame/liberland-legislation/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use pallet::*;
+
+#[frame_support::pallet]
+pub mod pallet {
+  use frame_support::pallet_prelude::*;
+  use frame_system::pallet_prelude::*;
+
+  #[pallet::pallet]
+  #[pallet::generate_store(pub(super) trait Store)]
+  #[pallet::without_storage_info]
+  pub struct Pallet<T>(_);
+
+  #[pallet::config]
+  pub trait Config: frame_system::Config {
+    type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+  }
+  #[pallet::event]
+  #[pallet::generate_deposit(pub(super) fn deposit_event)]
+  pub enum Event<T: Config> {
+    LawAdded { who: T::AccountId, index1: u32, index2: u32 },
+    LawRepealed { who: T::AccountId, index1: u32, index2: u32 },
+  }
+  #[pallet::error]
+  pub enum Error<T> {
+    LawAlreadyExists,
+  }
+  #[pallet::storage]
+  pub(super) type Laws<T: Config> = StorageDoubleMap<_, Blake2_128Concat, u32, Blake2_128Concat, u32, BoundedVec<u8, ConstU32<65536>>, ValueQuery>;
+  #[pallet::call]
+  impl<T: Config> Pallet<T> {
+
+    #[pallet::weight(0)]
```

Comment:
lets change this from 0 to 10000, as weight 0 will not charge any tx fee and will exhaust the system

Reply:
This is fine as only way to run this is through referendum. Not yet, but thats the idea

Reply:
This is in pallet-call trait == everyone can call the public function, please fix this and set this to 10000